python/cudf/cudf/tests/test_dataframe.py

```diff
@@ -3894,13 +3900,13 @@ def test_select_dtype_datetime_with_frequency():
 
 
 def test_dataframe_describe_exclude():
-    np.random.seed(12)
+    rng = np.random.default_rng(seed=0)
```

Comment:
Change of seed?

Reply:
Same reason as above, it didn't preserve the values we were previously getting/testing.

Reply:
Reverted to the original seed value.